scss/mixins/_pagination.scss

```diff
@@ -11,17 +11,66 @@
 
 // scss-docs-start pagination-max-items-mixin
 // Boosted mod
-// Ensure we only display `n` items:
-// - the first `n/2` items
-// - the last `n/2` items
-// - including the .active one (so maybe not the `n/2`th last item)
+// The pagination list ((m) items) should display exactly (n) items and is built in 3 parts:
+// - the begining of the list is made of the first (n/2) items
+// - the end of the list is made of the last (n/2) items (or (n/2 - 1) depending on the active element)
+// - the last part of the list is made of the active element
 @mixin pagination-max-items($pagination-max-items) {
-  display: none;
+  @if $enable-responsive-pagination {
+    $half-items: $pagination-max-items * .5;
+    display: none;
 
-  &.active,
-  &:nth-child(-n+#{$pagination-max-items * .5}),
-  &:nth-last-child(-n+#{$pagination-max-items * .5}) {
-    display: list-item;
+    // rule 0:
+    //  - adding the (n/2) first items
+    //  - adding the (n/2 - 1) last items
+    //  - adding the (n/2) last item if the list contains exactly (n) items
```

Comment:
So 'if the list contains exactly (n) items' equal 'all m items are displayed'?

Reply:
Yes exactly ! m = n here for this comment. Should I update ?